contribs/drt/src/main/java/org/matsim/contrib/drt/run/DrtConfigGroup.java

```diff
@@ -194,6 +194,13 @@ public enum OperationalScheme {
 	@PositiveOrZero
 	public double advanceRequestPlanningHorizon = 0; // beta-feature; planning horizon for advance (prebooked) requests
 
+	@Parameter
+	@Comment("When working with prebooked requests, defines whether vehicles drive immediately to the next"
+			+ "(prebooked) future task and wait for the planned stop to begin, or wait at the current"
+			+ "position and depart to arrive on time at the following stop. The latter behavior (not"
+			+ "the default) may lead to larger ucnertainty in highly congested scenarios.")
+	public boolean scheduleWaitBeforeDrive = false;
```

Comment:
I agree `false` should be the default.

Of course, in more advanced cases, there could be some threshold. For instance, a taxi would rather arrive 5 minute before the planned departure (rather than 30 minutes or 0 minutes).

Reply:
Yes, this is a good feature to note for a future version. Then this simply becomes a number instead of on/off.

Reply:
I'll add a comment in the code